src/v0/destinations/airship/utils.ts

```diff
@@ -49,27 +49,41 @@ type AirshipObjectAttributes = Partial<{
   removeAttributes: Omit<Attribute, 'value'>[];
 }>;
 
-const getDigitCount = (num: number): number => Math.floor(Math.log10(Math.abs(num))) + 1;
+type TimestampAttributes = Array<{
+  timestampAttribute: string;
+}>;
 
-export const isValidTimestamp = (timestamp: string | number): boolean => {
-  // Check if timestamp is a valid Unix timestamp (10 digits)
-  if (typeof timestamp === 'number' || !Number.isNaN(Number(timestamp))) {
-    return getDigitCount(Number(timestamp)) >= 10;
+export const convertToAirshipTimestamp = (timeValue: string | number): string => {
+  let millis;
+  let timestamp = timeValue;
+  // Check if the input is a string containing a numeric timestamp
+  if (typeof timestamp === 'string' && /^\d+$/.test(timestamp)) {
+    timestamp = Number(timestamp); // Convert string to number
   }
 
-  // Check if timestamp is a valid date string
-  const date = moment.utc(timestamp);
-  return date.isValid() && date.year() >= 1970;
-};
+  // Check if the input is a valid date string
+  if (typeof timestamp === 'string' && moment.utc(timestamp, moment.ISO_8601, true).isValid()) {
+    return moment.utc(timestamp).format(AIRSHIP_TIMESTAMP_FORMAT);
```

Comment:
Need to make a note in doc & UI, timestamp values are expected to be passed as UTC. If no timezone mentioned, will be considered as UTC